fileversion.go

```diff
@@ -169,6 +169,28 @@ type FixedFileInfo struct {
 	FileSubtype    VerFileSubtype
 }
 
+type StringFileInfoNames string
+
+const (
+	FileInfoNameComments         StringFileInfoNames = "Comments"
+	FileInfoNameCompanyName      StringFileInfoNames = "CompanyName"
+	FileInfoNameFileDescription  StringFileInfoNames = "FileDescription"
+	FileInfoNameFileVersion      StringFileInfoNames = "FileVersion"
+	FileInfoNameInternalName     StringFileInfoNames = "InternalName"
+	FileInfoNameLegalCopyright   StringFileInfoNames = "LegalCopyright"
+	FileInfoNameLegalTrademarks  StringFileInfoNames = "LegalTrademarks"
+	FileInfoNameOriginalFilename StringFileInfoNames = "OriginalFilename"
+	FileInfoNameProductName      StringFileInfoNames = "ProductName"
+	FileInfoNameProductVersion   StringFileInfoNames = "ProductVersion"
+	FileInfoNamePrivateBuild     StringFileInfoNames = "PrivateBuild"
+	FileInfoNameSpecialBuild     StringFileInfoNames = "SpecialBuild"
+)
+
+type FileInfoTranslation struct {
+	Language uint16
+	CodePage uint16
```

Comment:
This is one of very few places in the Win32 API where a code page is treated as a 16-bit integer as opposed to a 32-bit integer.  We probably want to expose it to users as a Go `uint`.